lib/ezQuery.php

```diff
@@ -428,12 +438,37 @@ private function processConditions($column, $condition, $value, $valueOrCombine,
         } elseif ((($condition == \_LIKE) || ($condition == \_notLIKE)) && !\preg_match('/[_%?]/', $value)) {
             return $this->clearPrepare();
         } else {
-            $this->conditions($column, $condition, $value, $valueOrCombine);
+            $this->conditions($column, $condition, $value, $valueOrCombine, $extraCombine);
+        }
+    }
+
+    public function whereGroup(...$whereConditions)
```

Comment:
What you think about just calling it `group` or `grouping`? 
The only place it can be used in is with `where` already. 
Repeating the same word within the same function call trying to stay away from, and keep naming somewhat similar to regular SQL dialect.

Reply:
Good idea. Have updated the code and tests. I didn't want to call it group as that felt too close to `groupBy` for me and I thought it might get confused, but I like `grouping`.